index.html

```diff
@@ -44,4 +44,117 @@ <h4>
       </ul>
     </nav>
   </section>
-  
\ No newline at end of file
+ 
+  <!-- Mobile Works Section -->
+  <section class="grid-container">
+    <div class="flex-cards" >
+      <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot">
+      <h3>Tonic</h3>  
+      <span>
+       Canopy
+        <img src="images/Counter.png" alt="circle"> Back End Dev
+        <img src="images/Counter.png" alt="circle"> 2015
+      </span>
+    
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img src="images/Tags.png" alt="Languages tag">
+      <a href="https://github.com/" class="button">See Project</a>
+    </div>
+    
+    
+    <div class="flex-cards">
+      <img src="images/Snapshoot-Portfolio-a.png" alt="Snapshoot">
+      <h3>Multi-post series</h3>
+      <span>
+        Canopy
+        <img src="images/Counter.png" alt="circle"> Back End Dev
+        <img src="images/Counter.png" alt="circle"> 2015
+      </span>
+    
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img src="images/Tags.png" alt="html tag">
+      <a href="https://github.com/" class="button">See Project</a>
+    </div>
+    
+    
+    <div class="flex-cards">
```

Comment:
- [ ] [OPTIONAL] Please let's make the website responsive also for `480px` since the mobile view actually should start from there.